tests/django/manage.py

```diff
@@ -29,4 +29,7 @@
 
     from django.core.management import execute_from_command_line
 
+    os.environ.setdefault(u'HYPOTHESIS_DJANGO_USETZ_TEST', u'TRUE')
```

Comment:
I think the main point to change is just the call from scripts/basic-test.sh. I'd just call it twice there with an environment variable set in each call.